pos_category_multi/README.rst

```diff
@@ -0,0 +1,27 @@
+==================
+POS Category Multi
+==================
+
+This module allows you to change the type of POS Categories field from many2one to many2many. Wherein, all existing Pos Categories category will be reset. This field is used in product.product (Product Variants) and product.template (Products).
```

Comment:
Delete

    This field is used in product.product (Product Variants) and product.template (Products).